try (try raise(RuntimeException) catch RuntimeException as e in 1) catch RuntimeException as e in 5